lib/src/layout/resizable_pane_notifier.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/foundation.dart';
+
+class ResizablePaneNotifier extends ValueNotifier<Map<Key, double>> {
+  /// Creates a [ValueNotifier] that wraps this [value] for a [ResizablePane]
```

Comment:
Move this line above the class declaration

Reply:
I don't think this line should be moved up, otherwise the constructor would have no documentation.

Reply:
Class documentation is more important than constructor documentation

Reply:
Couldn't we use the dart doc macro here?

https://pub.dev/packages/dartdoc#macros

Example:
https://github.com/flutter/flutter/blob/1d9032c7e1d867f071f2277eb1673e8f9b0274e3/packages/flutter/lib/src/material/button.dart#L133